packages/clerk-js/src/ui/userProfile/EditableListFieldRemoveCard.tsx

```diff
@@ -68,11 +66,17 @@ export const EditableListFieldRemoveCard: React.FC<EditableListFieldRemoveCardPr
             yourself.
           </p>
         )}
-        {type === 'external_account' && <p>You will no longer be able to sign in using this connected account.</p>}
+        {type === 'connected_account' && (
+          <p>Unlink your {label} account, and remove associated personal information.</p>
+        )}
         {type === 'web3_wallet' && <p>Remove the connection between this app and your Web3 wallet</p>}
       </div>
       <div className='cl-form-button-group'>
-        <Button onClick={updateFieldSubmit}>Remove {formattedType}</Button>
+        <Button onClick={updateFieldSubmit}>
+          {type === 'connected_account' || type === 'web3_wallet'
```

Comment:
🔧 Let's just pass the string as a prop to avoid internal conditionals as they create a bit of a code smell.